nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -128,6 +128,16 @@ in
         '';
       };
 
+      firewallOpenPorts = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Specifies whether the ports the SSH daemon listens on must
+          be enabled in the firewall. Remember to open at least some
+          of them.
```

Comment:
Could you be more specific as to why it might be useful to combine `services.openssh.listenAddresses` with this PR?
I must confess I don't really understand how to use `services.openssh.listenAddresses`: it sends RST packets (revealing that there's an SSH server running on the machine), and it is so easy to lock yourself out by just testing things, without any real way to go back.

In my situation (VPS + nixops), this PR would have enabled me to just logged via a "web console", and type "iptables -F" to re-enable everything, whereas `listenAddresses` forced me to write a new ssh configuration in the web console (with an incorrect keyboard layout and a terrible lag), and then to try and understand how to start an SSH server in the command line just be able to run nixops again.